regolith/helpers/l_contactshelper.py

```diff
@@ -0,0 +1,136 @@
+"""Helper for finding and listing contacts from the contacts.yml database.
+Prints name, institution, and email (if applicable) of the contact.
+"""
+import datetime 
+import dateutil
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    fuzzy_retrieval,
+)
+
+TARGET_COLL = "contacts"
+HELPER_TARGET = "l_contacts"
+
+def subparser(subpi):
+    subpi.add_argument("-n", "--name", help='finds contacts based off a list of names or name fragments', nargs = '+')
+    subpi.add_argument("-i", "--inst", help='finds contacts based off an institution or an institution fragment')
```

Comment:
as above.  Is this a single inst or is a list of insts allow?  If multiple args are specified, are you using AND or OR (but write it in plain english not mathy way)